kornia/geometry/transform/imgwarp.py

```diff
@@ -158,6 +158,68 @@ def warp_affine(src: torch.Tensor, M: torch.Tensor,
                          padding_mode=padding_mode)
 
 
+def warp_affine3d(src: torch.Tensor, M: torch.Tensor,
```

Comment:
@shijianjian this is already warp_projective, right ?

Reply:
Right. I will clean it up. I did not use those functions here.

Reply:
BTW, ```warp_perspective``` is not used for the 2D ```affine()```. Is there any specific reason for that?